src/components/Accordion/Accordion.vue

```diff
@@ -40,12 +40,19 @@ export default {
     title: {
       type: String,
       default: 'Accordion Title'
+    },
+    open: {
+      type: Boolean,
+      default: false
     }
   },
   data () {
     return {
       expanded: false
     };
+  },
+  created () {
+    this.expanded = this.open;
```

Comment:
Also, not sure the thought process behind `defaultOpen` versus `open` and how `open` ended up the winner, but at first it confused me trying to understand how it differed from `expanded` and why it was only being used to set `expanded` during the `created` lifecycle hook.  Now I see that it really is meant to represent if the accordion should default to open, rather than just be open at any time.  So maybe `defaultOpen` would have been the better name to go with?

Reply:
That's fair it is the more descriptive word. I suggested changing it to 'open' because I thought that was pretty intuitive ("oh I want my accordion to be open so I can pass that in") and in line with how other props are named in the project. It also doesn't default to open so 'defaultOpen' felt weird.

Reply:
I guess I would assume if there is a prop named `open` that I could use it from the parent to externally tell the Accordion whether to open or close in addition to the user clicking the toggle icon.  But the way this is written it would only allow me to do that initially and any subsequent changes to the `open` value wouldn't change the state of the Accordion.  So that's why I found it a tad confusing.  But if that naming and behavior is consistent with other components maybe it's fine as long as it's clearly documented, which I guess also needs to be done.  So this needs to be added to the .mdx file probably?

Reply:
Oh I see what you mean and I agree that 'open' makes more sense as an event name than a prop name.